app/components/Thread/index.jsx

```diff
@@ -141,7 +141,7 @@ const Thread = React.createClass({
                     email_subject: `[RhinoBird] ${LoginStore.getUser().realname} mentioned you in comment`,
                     email_body: `${LoginStore.getUser().realname} mentioned you in comment of <a href="${window.location.href.split('#')[0]}">${this.props.threadTitle}</a>
                     <br/><br/> ${comment}`,
-                    url: `window.location.pathname#${c.cid}`
+                    url: window.location.href.split('#')[0]+"#" + c.cid
```

Comment:
I got, it is small mistake. BTW, you can change it to

```
`${window.location.pathname}#${c.cid}`
```

I think it is better than `href.split`

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/template_strings